paddle/fluid/pybind/pybind.cc

```diff
@@ -401,6 +401,24 @@ All parameter, weight, gradient are variables in Paddle.
 
   py::class_<framework::Executor>(m, "Executor")
       .def(py::init<const platform::Place &>())
+      .def_static(
+          "prepare",
+          [](const ProgramDesc &pdesc, int block_id) -> void * {
+            return static_cast<void *>(Executor::Prepare(pdesc, block_id));
+          },
+          py::return_value_policy::reference)
+      .def_static("delete_prepared_ctx",
```

Comment:
This method is not needed.

We can just return `ExecutorPrepareContext` in `prepare` method, and make Python delete this object.

Reply:
Ok, I use python copy to return the `void *` now:

```c++
    /** Create a new copy of the returned object, which will be owned by
        Python. This policy is comparably safe because the lifetimes of the two
        instances are decoupled. */
    copy,
```

I am not sure how will Python manage this returned pointer, can you give some information?

Reply:
The default policy is OK